### PR TITLE
Added support for obfs4

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

**Comment by myleshorton** [comment]
Hey so @oxtoacart is the test failure expected because of how you changed the merge to make it more readable?

**Comment by myleshorton** [comment]
it's looking for that tor obfs4 package..

**Comment by oxtoacart** [comment]
@myleshorton That's correct.  The full PR with dependencies is here - https://github.com/getlantern/lantern/pull/3971

File: src/github.com/getlantern/flashlight/client/chained.go [hunk 1]
```diff
@@ -25,82 +22,50 @@ var ForceChainedProxyAddr string
...
 	Trusted bool
+
+	// PluggableTransport: If specified, a pluggable transport will be used
+	PluggableTransport string
+
+	// PluggableTransportSettings: Settings for pluggable transport
+	PluggableTransportSettings map[string]string
 }
 
 // Dialer creates a *balancer.Dialer backed by a chained server.
```
Comment (myleshorton) [inline src/github.com/getlantern/flashlight/client/chained.go]: Minor nit, and this is only related to this PR in the sense that with pluggable transports there's more for the code reader to think about, but I think a comment for deviceID would be helpful here @oxtoacart -- i.e. make it clear it's not related to pluggable transports at all but just identifies this device to the server.

Comment (oxtoacart) [inline src/github.com/getlantern/flashlight/client/chained.go]: Fixed.

**Pull request merged**